test/tools/WebListener/Controllers/ResumeController.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http.Headers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using mvc.Models;
+
+
+namespace mvc.Controllers
+{
+    public class ResumeController : Controller
+    {
+        private static Byte[] FileBytes = new Byte[]{1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20};
+
+        public async void Index()
+        {
+            SetResumeResponseHeaders();
+            string rangeHeader;
+            int from = 0;
+            int to = FileBytes.Length - 1;
+            if (TryGetRangeHeader(out rangeHeader))
+            {
+                var range = GetRange(rangeHeader);
+                if(range.From != null)
+                {
+                    from = (int)range.From;
+                }
+                if(range.To != null)
+                {
+                    to = (int)range.To;
+                }
+
+            }
+            else
+            {
+                Response.ContentType = "application/octet-stream";
+                Response.StatusCode = 200;
```

Comment:
Any reason not to use HttpStatusCode enum?

Reply:
`HttpResponse.StatusCode` is an int32 . There is apparently a `Microsoft.AspNetCore.Http.StatusCodes` https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.statuscodes?view=aspnetcore-2.0 that has static fields. I'll adjust to use that. 

```csharp
Response.StatusCode = StatusCodes.Status200OK
```

Reply:
fixed